pandas/io/tests/test_sql.py

```diff
@@ -1165,6 +1165,21 @@ def test_to_sql(self):
     def test_to_sql_empty(self):
         self._to_sql_empty()
 
+    def test_to_sql_single_dtype(self):
+        self.drop_table('single_dtype_test')
+        cols = ['A','B']
+        data = [('a','b'),
+                ('c','d')]
+        df = DataFrame(data,columns=cols)
+        df.to_sql('single_dtype_test',self.conn,dtype=sqlalchemy.TEXT)
+        meta = sqlalchemy.schema.MetaData(bind=self.conn)
+        meta.reflect()
+        sqltypea = meta.tables['single_dtype_test'].columns['A'].type
+        sqltypeb = meta.tables['single_dtype_test'].columns['B'].type
+        self.assertTrue(isinstance(sqltypea, sqlalchemy.TEXT))
+        self.assertTrue(isinstance(sqltypeb, sqlalchemy.TEXT))
+        self.drop_table('single_dtype_test')
```

Comment:
Can you put these tests next to the current test for the dtype argument? (to keep things a little bit organised in the test file, it is already difficult to navigate here :-))